lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022, XMOS Ltd, All rights reserved
+
+
+#include "MemCpyFn.hpp"
+#include "tensorflow/lite/c/common.h"
+#include "tensorflow/lite/kernels/internal/tensor_ctypes.h"
+#include "tensorflow/lite/kernels/kernel_util.h"
+#include "tensorflow/lite/micro/kernels/kernel_util.h"
+#include "xcore_custom_options.h"
+#include "xcore_interpreter.h"
+#include "xcore_dispatcher.h"
+#include "xcore_utils.h"
+extern "C" {
+#include "nn_operator.h"
+}
+#include <cstdio>
```

Comment:
Please remove the headers we are not using. I think many can be removed.

Reply:
Removed